Formula/ansible.rb

```diff
@@ -59,40 +60,24 @@ class Ansible < Formula
   end
   ### end
 
-  resource "Babel" do
```

Comment:
poet really needs to handle this better; documented it at https://github.com/tdsmith/homebrew-pypi-poet/issues/43.